ext/readline/readline.c

```diff
@@ -307,6 +315,22 @@ PHP_FUNCTION(readline_info)
 			RETVAL_STRING(SAFE_STRING(rl_prompt));
 		} else if (!strcasecmp(what, "terminal_name")) {
 			RETVAL_STRING((char *)SAFE_STRING(rl_terminal_name));
+		} else if (!strcasecmp(what, "completion_suppress_append")) {
+			oldval = rl_completion_suppress_append;
```

Comment:
According to https://tiswww.case.edu/php/chet/readline/CHANGES this variable has been added in readline 4.3, which according to https://lists.gnu.org/archive/html/info-gnu/2002-07/msg00006.html has been released in 2002. So using this without further checks should be fine.